src/modules/perl.rs

```diff
@@ -0,0 +1,210 @@
+use super::{Context, Module, RootModuleConfig, SegmentConfig};
+
+use crate::configs::perl::PerlConfig;
+use crate::utils;
+
+/// Creates a module with the current perl version
+///
+/// Will display the perl version if any of the following criteria are met:
+///     - Current directory contains a `.pl` or a `.pm` file
+///     - Current directory contains a "Makefile.PL", "cpanfile", "META.json", "META.yml", or ".perl-version" file
+pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
+    let is_perl_project = context
+        .try_begin_scan()?
+        .set_files(&[
+            "Makefile.PL",
+            "Build.PL",
+            "cpanfile",
+            "cpanfile.snapshot",
+            "META.json",
+            "META.yml",
+            ".perl-version",
+        ])
+        .set_extensions(&["pl", "pm", "pod"])
+        .is_match();
+
+    if !is_perl_project {
+        return None;
+    }
+
+    let perl_version = utils::exec_cmd("perl", &["-e", "printf q#%vd#,$^V;"])?.stdout;
```

Comment:
```suggestion
    let perl_version = utils::exec_cmd("perl", &["-e", "'printf q#%vd#,$^V;'"])?.stdout;
```
Note the inner single quotes, without them tests won't get mocked but real Perl version.

Reply:
ok done

Reply:
thanks for you help this is my first PR in rust

Reply:
And now all check pass, not just ubuntu, cool.

Reply:
Let me tell you a _secret_, I mostly do node 🤣

Reply:
There is a regression with this modification
![image](https://user-images.githubusercontent.com/4223252/85589837-31566b80-b644-11ea-82e0-3da992003cc2.png)

Reply:
resolved by 916e1b1

Reply:
Actually I was right back then, this should be quoted. The reason it failed is that mock also needed adjustments.

Reply:
nope. mock has not failed it's some other module. with quotes it fails